lib/src/test/java/vmath/Mat4Test.java

```diff
@@ -36,4 +36,22 @@ void givenInvalidIndices_whenGetCalled_thenThrowIndexOutOfBoundsException() {
         assertThrows(IndexOutOfBoundsException.class, () -> m.get(0, -1));
         assertThrows(IndexOutOfBoundsException.class, () -> m.get(0, 4));
     }
+
+    @Test
+    void givenFloatArray_whenMat4Created_thenMatrixMatchesArray() {
+        float[] mat = {
+                1.0f, 2.0f, 3.0f, 4.0f,
+                5.0f, 6.0f, 7.0f, 8.0f,
+                9.0f, 10.0f, 11.0f, 12.0f,
+                13.0f, 14.0f, 15.0f, 16.0f
```

Comment:
Follow the style specified in our dev. [docs](https://github.com/toksaitov/vmath/wiki/Code-Style).